index.html

```diff
@@ -433,6 +460,17 @@ <h2 class='mb-0'>Gato-Bots</h2>
                       <rect class='fill' fill='#fff' x='180' y='220' width='20' height='10' />
                     </svg>
                   </div>
+                  <div class='carousel-item'>
+                    <svg viewBox='0 160 320 80' width='100%'>
+                      <rect class='fill' fill='#fff' x='139' y='170' width='12' height='30' />
+                      <rect class='fill' fill='#fff' x='135' y='210' width='20' height='10' />
+                      <rect class='accent-fill' x='143' y='210' width='4' height='10' />
+
+                      <rect class='fill' fill='#fff' x='169' y='170' width='12' height='30' />
+                      <rect class='fill' fill='#fff' x='165' y='210' width='20' height='10' />
+                      <rect class='accent-fill' x='173' y='210' width='4' height='10' />
+                    </svg>
```

Comment:
same deal with the legs, each block is a leg + foot + hoof mark (the accent-fill drawn on top of the foot)